bin/dev

```diff
@@ -90,11 +90,18 @@ runJstest(){
 # Run precommit inside container
 runPrecommit(){
     findBackendContainerId container_id
-    cmd="docker exec -t -w /home/worker/app/ "$container_id" pre-commit "$*
+    # if no cli args, run standard pre-push
+    if [ ! $* ]; then
+    args="run --hook-stage push --from-ref origin/main --to-ref HEAD"
```

Comment:
pre-commit confirmed working on pre-push
Set it to compare between your branch and origin/main
Best practice is to always branch off of main anyways so this should be fine.

Reply:
How does this work with gh stash, does it still compare off of main?

Reply:
yep ghstack still compares everything against main - usually you're on the same branch and every commit is a pr so it should be fine - unless you have an absolutely massive stack of prs that touch every file in the repo - in which case it'll just take a while to run.